packages/driver/src/dom/dimensions.ts

```diff
@@ -4,16 +4,39 @@
 
 import _ from 'lodash'
 
-const getElementDimensions = ($el) => {
-  const el = $el.get(0)
+interface Box {
+  offset: { top: number, left: number }
+  paddingTop: number
+  paddingRight: number
+  paddingBottom: number
+  paddingLeft: number
+  borderTop: number
+  borderRight: number
+  borderBottom: number
+  borderLeft: number
+  marginTop: number
+  marginRight: number
+  marginBottom: number
+  marginLeft: number
+  width?: number
+  height?: number
+  heightWithPadding?: number
+  heightWithBorder?: number
+  heightWithMargin?: number
+  widthWithPadding?: number
+  widthWithBorder?: number
+  widthWithMargin?: number
+}
+
+const getElementDimensions = ($el: any) => {
+  const el: HTMLElement = $el.get(0)
 
   const { offsetHeight, offsetWidth } = el
 
-  const box = {
+  const box: Box = {
     // offset disregards margin but takes into account border + padding
     offset: $el.offset(),
     // dont use jquery here for width/height because it uses getBoundingClientRect() which returns scaled values.
-    // TODO: switch back to using jquery when upgrading to jquery 3.4+
```

Comment:
I don't think this TODO would be wise actually. The logic jQuery is using in later versions to get dimensions does not agree with what we want - we've had to patch over it to get back to <3.4 behavior, so I don't think this should be a TODO.